lib/data/firebase_auth/firebase_auth_manager.dart

```diff
@@ -1,68 +0,0 @@
-import 'package:book/models/app_user.dart';
```

Comment:
why you deleted this class?

Reply:
I put all of content from that  file into firebase_auth_singleton and just deleted that class.